### PR TITLE
perf(nuxt): experimental support of `capo.js` head tag order

### PR DESCRIPTION
**Pull request by harlan-zw**
### 🔗 Linked issue

https://github.com/danielroe/nuxt-capo/issues/36#issuecomment-1657240168

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [x] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Unhead 1.2.0 introduces an optional [CAPO plugin](https://github.com/unjs/unhead/blob/main/packages/unhead/src/plugin/capoPlugin.ts). We allow a simple opt-in for this in Nuxt.

```ts
export default defineNuxtConfig({
  experimental: {
    headCapoPlugin: true
  }
})
```

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by danielroe — APPROVED** [review]

File: packages/nuxt/src/head/runtime/plugins/capo.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { CapoPlugin, injectHead } from '@unhead/vue'
+import { defineNuxtPlugin } from '#app/nuxt'
+
+export default defineNuxtPlugin({
+  name: 'nuxt:head:capo',
+  setup () {
+    injectHead().use(CapoPlugin)
```
Comment (danielroe) [inline packages/nuxt/src/head/runtime/plugins/capo.ts:7]:
```suggestion
    injectHead().use(CapoPlugin())
```

File: packages/schema/src/config/experimental.ts [hunk 2]
```diff
@@ -207,6 +207,13 @@ export default defineUntypedSchema({
      * @see https://github.com/parcel-bundler/watcher
      * @type {'chokidar' | 'parcel' | 'chokidar-granular'}
      */
-    watcher: 'chokidar-granular'
+    watcher: 'chokidar-granular',
+
+    /**
+     * Add the CAPO head plugin in order to render tags in of the head in a more performant way.
+     *
+     * @default false
```
Comment (danielroe) [inline packages/schema/src/config/experimental.ts:215]:
```suggestion
```

**Comment by ronenteva** [comment]
Is this now implemented? Removed? Or just missing from https://nuxt.com/docs/guide/going-further/experimental-features?

**Comment by manniL** [comment]
@ronenteva This is now the [`headNext` flag](https://nuxt.com/docs/api/nuxt-config#headnext).

**Pull request merged**